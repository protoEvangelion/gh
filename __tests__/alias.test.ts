/**
 * Â© 2013 Liferay, Inc. <https://liferay.com> and Node GH contributors
 * (see file: CONTRIBUTORS)
 * SPDX-License-Identifier: BSD-3-Clause
 */

import { runCmd } from './testUtils'

describe('E2E: Alias Module Test', () => {
    it('User alias is expanded `gh user --user=zeno`', done => {
        expect(runCmd('gh user --user=zeno')).toMatchSnapshot()
        done()
    })
})
