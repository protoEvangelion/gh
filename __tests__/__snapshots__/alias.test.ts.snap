// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E: Alias Module Test User alias is expanded \`gh user --user=zeno\` 1`] = `
"You're logged in as zenorocha
"
`;
